### PR TITLE
Angular cypress test

### PR DESCRIPTION
**Pull request by HarelM**
This is a single test and It's not complete, I just want to see if the CI works... :-)

### CODE REVIEW

**Comment by HarelM** [comment]
No CI for PRs :-(((
This is not ready to be merged BTW...

File: examples/angular/pokemon-catalog/src/app/components/pokemon-go/pokemon-go.component.spec.cy.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+describe("Angular PokemonGoComponent Tests", () => {
+  const testConfig = {
+    declarations: [PokemonGoComponent]
+  };
+
+  const { when, get, beforeAndAfter } =
+    new PokemonGoComponentDriver();
+  beforeAndAfter();
+
+  it("given input filled should have input value", async () => {
```
Comment (ShellyDCMS) [inline examples/angular/pokemon-catalog/src/app/components/pokemon-go/pokemon-go.component.spec.cy.ts:13]: when, not given

File: examples/angular/pokemon-catalog/src/app/components/pokemon-go/pokemon-go.component.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<div>
+    <input data-cy="search-input" type="number" [(ngModel)]="searchTerm" />
+    <button data-cy="go" (click)="onSumbit()">Go</button>
+</div>
```
Comment (ShellyDCMS) [inline examples/angular/pokemon-catalog/src/app/components/pokemon-go/pokemon-go.component.html:4]: add newline

File: examples/angular/pokemon-catalog/src/app/components/pokemon-catalog/pokemon-catalog.component.ts [hunk 3]
```diff
@@ -56,4 +56,8 @@ export class PokemonCatalogComponent {
         .filter(element => element)
         .pop()
     );
+  
+  async selectPokemon(index: number) {
```
Comment (ShellyDCMS) [inline examples/angular/pokemon-catalog/src/app/components/pokemon-catalog/pokemon-catalog.component.ts:60]: arrow function please ...

File: examples/angular/pokemon-catalog/src/app/components/pokemon-go/pokemon-go.component.spec.cy.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    when.render(PokemonGoComponent, testConfig);
+    when.typePokemonIndex("42");
+    expect(await get.selectedPokemon()).to.eq("42");
+  });
+
+  it("given input and clicked go should get event value", () => {
```
Comment (ShellyDCMS) [inline examples/angular/pokemon-catalog/src/app/components/pokemon-go/pokemon-go.component.spec.cy.ts:19]: same

File: examples/angular/pokemon-catalog/src/app/components/pokemon-go/pokemon-go.component.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    @Output()
+    selectedPokemon: EventEmitter<number> = new EventEmitter<number>();
+
+    public searchTerm: number | undefined = undefined;
+
+    public onSumbit() {
+        this.selectedPokemon.emit(this.searchTerm);
+    }
+
+}
```
Comment (ShellyDCMS) [inline examples/angular/pokemon-catalog/src/app/components/pokemon-go/pokemon-go.component.ts:18]: new line

**Review by ShellyDCMS — COMMENTED** [review]
Can you add an integration test please?

**Comment by HarelM** [comment]
Done. Added an integration test, updated code with review.

File: examples/angular/pokemon-catalog/src/app/components/pokemon-go/pokemon-go.component.ts (hunk 5 above)
Comment (HarelM) [reply examples/angular/pokemon-catalog/src/app/components/pokemon-go/pokemon-go.component.ts]: Done.

**Pull request merged**